packages/go_router/lib/src/typedefs.dart

```diff
@@ -2,22 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
```

Comment:
Is this change necessary?

```suggestion
import 'package:flutter/widgets.dart';
```